app/editor/src/components/layout/DefaultLayout.tsx

```diff
@@ -31,7 +30,11 @@ export const DefaultLayout: React.FC<ILayoutProps> = ({ name, children, ...rest
       <UserInfo />
       {keycloak.authenticated ? (
         <>
-          <Header name={name} />
+          <Header name={name}>
+            <Button onClick={() => keycloak.instance.logout()} name="signOut">
```

Comment:
Makes it possible to have a single Header component without throwing an error when keycloak hasn't been initialized yet.